libsel4/include/sel4/bootinfo_types.h

```diff
@@ -79,11 +80,26 @@ typedef struct seL4_BootInfo {
      * to make this struct easier to represent in other languages */
 } seL4_BootInfo;
 
-/* If extraLen > 0, then 4K after the start of bootinfo there is a region of the
- * size extraLen that contains additional boot info data chunks. They are
- * arch/platform specific and may or may not exist in any given execution. Each
- * chunk has a header that contains an ID to describe the chunk. All IDs share a
- * global namespace to ensure uniqueness.
+/* The boot info frame must be large enough to hold the seL4_BootInfo data
+ * structure. Due to internal restrictions, the size must be of the form 2^n and
+ * the minimum is one page.
+ */
+#define seL4_BootInfoFrameSizeBits  seL4_PageBits
```

Comment:
I think this name should just be `seL4_BootInfoFrameBits` to follow the Size/Bits naming pattern that the other constants follow. (Thoughts @lsf37?)

Reply:
Yes, makes sense. Will change this.

Reply:
Thanks!